elements/noflo-ui.html

```diff
@@ -316,7 +319,23 @@
         if (context.db) {
           this.$.journal.db = context.db;
         }
+        if (context.persisted) {
+          if (context.persisted.type === 'runtimes') {
+            // For now we only handle runtimes through action loop
```

Comment:
Down the line we should do this in a reducer instead of the polymer element, but those parts of #680 will be extracted to a PR of their own